Description: disable broken doctest
 The doctest in question fails to build, I tried to fix it following
 https://stackoverflow.com/questions/31644207/cannot-find-macro-error-in-the-macros-own-doc-test
 but failed to work out the correct incantation to import the macro.
 .
 Note that doctests are only run during build-time tests not autopkgtests
 due to the bahaviour of --all-targets, see https://github.com/rust-lang/cargo/issues/6669
 .
 This particular doctest was also not run previously at build time, I
 am not sure what changed to cause it to be run.
Author: Peter Michael Green <plugwash@debian.org>
Bug-Debian: https://bugs.debian.org/1026663
Last-Update: 2022-12-20

--- rust-sequoia-openpgp-mt-0.1.0.orig/src/macros.rs
+++ rust-sequoia-openpgp-mt-0.1.0/src/macros.rs
@@ -66,7 +66,7 @@ macro_rules! tracer {
 /// displays the output on stderr.  The output is prefixed with label,
 /// if it is provided.
 ///
-/// ```
+/// ```ignore
 /// let result = time_it!("Some code", 10, {
 ///     // Some code.
 ///     5
